buildSrc/src/main/kotlin/net/skyscanner/backpack/tokens/BpkTextUnits.kt

```diff
@@ -102,10 +105,15 @@ private fun toCompose(
           .builder(CaseFormat.UPPER_UNDERSCORE.to(CaseFormat.UPPER_CAMEL, name), TextUnitClass)
           .addModifiers(if (internal) KModifier.INTERNAL else KModifier.PUBLIC)
           .initializer(buildCodeBlock {
+            val extension = if (namespace == "BpkLetterSpacing") {
```

Comment:
I'm not happy with this, but this is the only difference between letter spacing & the other text tokens. I originally thought of adding an enum for the type, but as this ultimately gets called from the `tokens.gradle.kts` file it's not a very nice solution either